Index: src/ToDoList/ToDoList.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import React from \"react\";\r\nimport './ToDoList.css';\r\nimport \"antd/dist/antd.css\";\r\nimport { Button, Card, Checkbox } from \"antd\";\r\nimport ToDoItem from './components/ToDoItem/ToDoItem';\r\n\r\n\r\nconst ToDoList = () => {\r\n\r\n\r\n    return (\r\n        <div className=\"ToDoList\">\r\n            <div className=\"ContainerForTickets\">\r\n                <ToDoItem />\r\n                <Card style={{ width: 300 }}>\r\n                    <p>Card content</p>\r\n                </Card>\r\n            </div>\r\n\r\n            <Button type=\"primary\">Creat to do</Button>\r\n        </div>\r\n    );\r\n\r\n}\r\n\r\nexport default ToDoList;
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/ToDoList/ToDoList.tsx b/src/ToDoList/ToDoList.tsx
--- a/src/ToDoList/ToDoList.tsx	(revision 12593d44475d3ce1342877f4fbd26b5d3c26cedc)
+++ b/src/ToDoList/ToDoList.tsx	(date 1630668730298)
@@ -12,11 +12,7 @@
         <div className="ToDoList">
             <div className="ContainerForTickets">
                 <ToDoItem />
-                <Card style={{ width: 300 }}>
-                    <p>Card content</p>
-                </Card>
             </div>
-
             <Button type="primary">Creat to do</Button>
         </div>
     );
